examples/gatekeeper/unique-names/suite.yaml

```diff
@@ -0,0 +1,21 @@
+kind: Suite
+apiVersion: test.gatekeeper.sh/v1alpha1
+metadata:
+  name: block-duplicate-names
+tests:
+- name: block-duplicate-names
+  template: template.yaml
+  constraint: samples/constraint.yaml
+  cases:
+  - name: have duplicate name
+    object: samples/duplicate_name.yaml
+    inventory:
+    - samples/duplicate_name_data.yaml # This contains a sample to mock an already existing object
```

Comment:
Add another existing object to show it works with multiple objects and rego's implicit iteration.

Reply:
adding another item in the inventory as is it didn't made sense, so I added a more realistic scenarios that will allow you to have multiple resources pointing to the same AWS resources, as long they are observe only, but if there is already a xr resource in not observe, and the reviewed object is not observe only it will not be able avoiding having two claims trying to update the same AWS resource, or letting the user know early that's something we don't want to allow to create the claims like that